core/test/base/exception.cpp

```diff
@@ -44,7 +44,6 @@ TEST(ExceptionClasses, NotSupportedReturnsCorrectWhatMessage)
         error.what());
 }
 
-
```

Comment:
accident?

Reply:
argh, yes, the double spacing gets me a lot 😞